packages/teams-js/src/public/runtime.ts

```diff
@@ -97,6 +101,9 @@ export const teamsRuntimeConfig: IRuntime = {
       update: {},
     },
     logs: {},
+    location: {
```

Comment:
Do we need to make a change to versionContants below to take into account the addition of the 'map' subcapability?